### ISSUE TITLE
At which steps training should complete for rnet and onet??

### ISSUE THREAD

**Issue by ayush431** [wangbm/MTCNN-Tensorflow#24]

**Comment by galuhputraa** [comment]
im already in this step for pnet?
Step 808673 for cls: loss = 0.15127. Step 808526 for bbx: loss = 0.00650. Step 1 for pts: loss = 0.00000

**Comment by galuhputraa** [comment]
PNet: Step 1547385 for cls: loss = 0.13956. Step 1548614 for bbx: loss = 0.00630. Step 1 for pts: loss = 0.00000
Done training for 200 epochs, 1547408 steps. about 3 hour

RNet:
ONet:
